src/test/scala/org/ergoplatform/nodeView/wallet/ErgoWalletSpec.scala

```diff
@@ -617,6 +619,41 @@ class ErgoWalletSpec extends PropSpec with WalletTestOps {
     }
   }
 
+  property("only certain and unspent boxes is used for transaction generation") {
```

Comment:
This test pass with old wallet storage. It should fail with old WalletStorage, and pass with the new one

Reply:
@catena2w, already fixed.